tests/functional/spec/census/issues/household-member-details-q1-are-these-details-correct.spec.js

```diff
@@ -0,0 +1,106 @@
+import chai from 'chai'
+import {openAndStartCensusQuestionnaire} from '../../../helpers'
+
+import PermanentOrFamilyHome from '../../../pages/surveys/census/household/permanent-or-family-home.page.js'
```

Comment:
It looks like many of these imports aren't actually used. I count 17 pages in the tests, with 71 imports. Can they be tidied up?